Fix:
/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk/usr/include/mach/lock_set.h:75:15: error: conflicting types for 'neko_lock_acquire'
/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk/usr/include/mach/lock_set.h:87:15: error: conflicting types for 'neko_lock_release'
/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk/usr/include/mach/lock_set.h:99:15: error: conflicting types for 'neko_lock_try'
Part of:
https://github.com/HaxeFoundation/neko/commit/7170d2c5b83076fc605704a20795272281f23508
--- libs/ssl/ssl.c.orig
+++ libs/ssl/ssl.c
@@ -16,7 +16,10 @@ typedef int SOCKET;
 #endif
 
 #ifdef NEKO_MAC
-#include <Security/Security.h>
+#include <CoreFoundation/CoreFoundation.h>
+#include <Security/SecKeychain.h>
+#include <Security/SecItem.h>
+#include <Security/SecCertificate.h>
 #endif
 
 #define SOCKET_ERROR (-1)
